src/DependencyResolver/Package.php

```diff
@@ -25,7 +30,11 @@
  */
 final class Package
 {
-    /** @param list<ConfigureOption> $configureOptions */
+    /**
+     * @param list<ConfigureOption>       $configureOptions
+     * @param list<OperatingSystemFamily> $compatibleOsFamilies
```

Comment:
Hey @alexandre-daubois please can you check out some suggested improvements alexandre-daubois/pie#2